virttest/nfs.py

```diff
@@ -162,7 +162,8 @@ def __init__(self, params):
             self.nfs_setup = True
             path.find_command("service")
             path.find_command("exportfs")
-            if distro_details.name == 'Ubuntu':
+            distro_name = distro_details.name.lower()
```

Comment:
Suggest still keep distribution name as before, and no need to be converted to lower case, For RHEL, please check related distribution name

Reply:
fixed.